metrics/examples/prometheus/install/prometheus-rules.yaml

```diff
@@ -33,6 +33,14 @@ spec:
       annotations:
         summary: 'Kafka abnormal controller state'
         description: 'There are {{ $value }} active controllers in the cluster'
+    - alert: OfflinePartitions
+      expr: sum(kafka_controller_kafkacontroller_offlinepartitionscount) > 0
+      for: 10s
+      labels:
+        severity: warning
+      annotations:
+        summary: 'Kafka offline partitions'
+        description: 'There are offline partitions on pod {{ $labels.kubernetes_pod_name }}/{{ $labels.instance }}'
```

Comment:
What does offline partition mean? Is it when all replicas for given partition are down? In that case, they will be most likely on several pods, so does the message make sense here?